AuthViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  AuthViewController.swift
+//  Shapify
+//
+//  Created by Alex on 25/12/2022.
+//
+
+import UIKit
+import WebKit
+
+class AuthViewController: UIViewController {
```

Comment:
I think we might even make it final class, since this controller probably won't be inherited.